codegen-plugin/src/main/kotlin/io/spine/protodata/hello/BuilderExtensionGenerator.kt

```diff
@@ -86,6 +93,11 @@ internal class BuilderExtensionGenerator(
             val functionName = "validate" + fieldName.camelCase() + "Count"
             val builderClass = fullClassName.nestedClass("Builder")
 
+            val validationMethod = "$javaPackage.$simpleTypeName" +
+                    "BuilderExtKt.$functionName(this);"
```

Comment:
This expression reads weird to me. It seems to render something like this:
```
com.acme.MySimpleTypeNameBuilderExtKt.validateFunctionName(this)
```
It's if I try to translate it into a string it with some arbitrary values. See there is no `.` before `BuilderExtKt`?